### PR TITLE
Add option `matlab_short_links`

### PR DESCRIPTION
**Pull request by joeced**
Working on getting more MATLAB like rendering and links.

Rather than `target.+package.ClassBar`, it will displayed and referenced as `package.ClassBar`.

### CODE REVIEW

**Comment by rdzman** [comment]
Beautiful! I started working on this at the end of last week, but you got much further much more quickly than me. I'll play around with this a bit more and comment as soon as I have anything. But, assuming this works consistently as intended, I'd vote to release it ASAP, as it's a huge step forward. Thanks!

**Comment by joeced** [comment]
A short progress report. It's slow and steady getting closer. I can now render objects in the root folder 🥳

billede

However, I still have a few details to iron out around base class linking.

**Comment by rdzman** [comment]
This is great. Thanks. I've been looking at some of this code with a view to starting work on #178. And I agree that you're making that easier.

File: sphinxcontrib/matlab.py [hunk 1]
```diff
@@ -830,8 +849,9 @@ def setup(app):
     # autodoc
     app.add_config_value("matlab_src_dir", None, "env")
     app.add_config_value("matlab_src_encoding", None, "env")
-    app.add_config_value("matlab_keep_package_prefix", True, "env")
+    app.add_config_value("matlab_keep_package_prefix", False, "env")
     app.add_config_value("matlab_show_property_default_value", False, "env")
+    app.add_config_value("matlab_short_links", False, "env")
```
[the remark this answers is not in the document]
Comment (joeced) [reply sphinxcontrib/matlab.py:865]: Should be fixed now

**Comment by joeced** [comment]
@rdzman I do think it's ready for merging to master now :)

**Pull request merged**